### DISCUSSION TITLE
Any ETA for DevTool 2.0?

### DISCUSSION THREAD

**Discussion by itayasaf1990xd** [DevToys-app/DevToys#1064] [General]
Any ETA for DevTool 2.0?

**Comment by veler** [comment]
We were hoping to release a preview at the end of this month. But realistically it will likely be in April or May.
It remains us 4 tools to migrate from DevToys 1.0 to 2.0, along with finishing some touch up (which are in progress) and prepare release material (installers, website...etc), which we didn't start yet.
